perl/lib/Sanger/CGP/Pindel/OutputGen/BamUtil.pm

```diff
@@ -236,30 +234,22 @@ sub parse_samples{
 }
 
 
-sub _compare_name_n_length{
+sub _compare{
 	my($contig,$contig1) = @_;
 	croak 'Incorrect input' unless ref $contig eq 'Sanger::CGP::Vcf::Contig';
 
   # check for defined mismatch
   return 0 if((defined $contig->name      ? 1 : 0) != (defined $contig1->name     ? 1 : 0));
   return 0 if((defined $contig->length    ? 1 : 0) != (defined $contig1->length   ? 1 : 0));
+  return 0 if((defined $contig->checksum  ? 1 : 0) != (defined $contig1->checksum ? 1 : 0));
```

Comment:
Both files don't have to have a checksum, this forces both to have it defined.

Reply:
Modified logic to handle undefined checksum